### DISCUSSION TITLE
Not sure how to wire the screen to make tft_espi touch work

### DISCUSSION THREAD

**Discussion by PLVaisman** [Bodmer/TFT_eSPI#2684] [Q&A - Touch]
1. I'm trying to make the touch function of the screen work. Actually the screen with the library itself is working well 
2. I'm using PlatformIO
3. TFT_eSPI library version bodmer/TFT_eSPI@^2.5.30
5. ESP32 S3 
6. https://wiki.dfrobot.com/3.5inches_480_320_TFT_LCD_Capacitive_Touchscreen_SKU_DFR0669#target_3
7. Interface type: SPI

Plus further information as appropriate to the problem:
1. 
 SCLK: 14
 MOSI: 13
MISO: 12
DC: 16
CS: 15
RST: 17
BL: 21
SCL: 38
SDA: 39
INT: 40
   
The code used is the Touch Controller Demo in your examples
And the user_setup:
```
#define ILI9488_DRIVER  
  #define TFT_MISO 12
#define TFT_MOSI 13
#define TFT_SCLK 14
#define TFT_CS   15  // Chip select control pin
#define TFT_DC   16  // Data Command control pin
#define TFT_RST  17  // Reset pin (could connect to RST pin)
#define TOUCH_CS 39
```
![df](https://github.com/Bodmer/TFT_eSPI/assets/132054744/83a16f77-f23e-4028-ac04-bf19a40190e1)
![df1](https://github.com/Bodmer/TFT_eSPI/assets/132054744/1ee79962-bb6a-44fd-b092-eaadb9e5f6b1)
![df2](https://github.com/Bodmer/TFT_eSPI/assets/132054744/1cc104f5-d3c3-4392-8737-83d34464751b)

What am i doing wrong ?

Best regards

**Comment by Bodmer** [comment]
The library provides support for XPT2046 resistance based touch controllers. Capacitance based screens require a separate library.

**Comment by TobiasVanDyk** [comment]
Will this help?
https://github.com/Raycast7373/FreeTouchDeck
ESP32-3248S035 Capacitive / GT911 touchscreen users
Make sure to uncomment the line //#define esp323248s035c if you use a ESP32-3248S035 Capacitive or //#define GT911 If you have a GT911 touchscreen

Also:  TAMCTec GT911 Library (version 1.0.2), https://github.com/TAMCTec/gt911-arduino
